feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/item/SpeakerItem.kt

```diff
@@ -71,7 +71,7 @@ class SpeakerItem @AssistedInject constructor(
         other as SpeakerItem
 
         if (speaker != other.speaker) return false
-        if (query == null || other.query == null || query != other.query) return false
+        if (query != other.query) return false
```

Comment:
It seems `query == null && other.query == null` should be false in the previous code but this doesn't. Is it okay?

Reply:
Previous code was wrong. Because GroupAdapter updated SpeakerItem even if it have same contents. (SpeakerItem looked blinking with the previous code)